tidb_row_checksum(): Update docs (#17633)
pingcap/docs: system-variables.md

</CustomContent>

> - The [`TIDB_ROW_CHECKSUM()`](/functions-and-operators/tidb-functions.md#tidb_row_checksum) function can be used to get the checksum of a row.

Reviewer: ```suggestion
- You can use the [`TIDB_ROW_CHECKSUM()`](/functions-and-operators/tidb-functions.md#tidb_row_checksum) function to get the checksum of a row.
```

> - You can use the [`TIDB_ROW_CHECKSUM()`](/functions-and-operators/tidb-functions.md#tidb_row_checksum) function to get the checksum of a row.

Reviewer 2: ```suggestion
- You can use the [`TIDB_ROW_CHECKSUM()`](/functions-and-operators/tidb-functions.md#tidb_row_checksum) function to get the checksum value of a row.
```
